source/sass/blocks/extra.scss

```
@@ -23,50 +28,67 @@
   font-weight: normal;
 
   @media (min-width: $tablet-width) {
-    margin-bottom: 80px;
-    padding-left: 69px;
+    position: static;
+    margin-bottom: 81px;
+    padding-left: 0;
     padding-right: 39px;
     font-size: 36px;
     line-height: 36px;
+    max-width: 1440px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
   }
 
   @media (min-width: $desktop-width) {
-    grid-column: 1/-1;
-    padding-left: 0;
+    margin-bottom: 78px;
+    padding-left: 71px;
+    padding-right: 110px;
   }
 
-  &::after {
-    @include pseudo-element(100vh, 0);
-    z-index: -1;
-    top: 50%;
-    border-bottom: 2px solid $green;
-  }
-  //green line
-  &::before {
+  & span {
```

```suggestion
  span {
```

парент селектор лишний, тк итак они сделаются как .extra__title span